gilded-rose-llc-frontend/components/item-block.test.tsx

```diff
@@ -0,0 +1,27 @@
+import ItemBlock from './item-block'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+describe('ItemBlock', () => {
+  it('Should display the item name', () => {
+    const item = {
+      name: 'aged brie',
+      price: '50',
+    }
+    render(<ItemBlock name={item.name} price={item.price} />)
+    const name = screen.getByText(item.name)
+
+    expect(name).toBeInTheDocument()
+  })
+
+  it('Should display the item price', () => {
+    const item = {
+      name: 'aged brie',
+      price: '50',
+    }
+    render(<ItemBlock name={item.name} price={item.price} />)
+    const price = screen.getByText(item.price)
+
+    expect(price).toBeInTheDocument()
+  })
```

Comment:
Question: Do we need a test for the add to cart button?

Reply:
At first, I was thinking of leaving that for when we have that button actually do something but now I realize that is a pretty mission-critical piece of this page so yes it should be tested that it is rendered on the page.

Unit test for this was added in [f4d96ef](https://github.com/jramnani/GildedRoseJavaLLC/pull/32/commits/f4d96ef89dbc119e5fa92d94a6b178ce40227647)